data/map.txt

```diff
@@ -12719,32 +14341,337 @@ system Hunter
 	fleet "Large Saryd" 1200
 	fleet Heliarch 900
 	object
-		sprite star/k5
+		sprite star/k5
+		period 10
+	object
+		sprite planet/cloud6
+		distance 256.01
+		period 79.4787
+	object "Shadow of Leaves"
+		sprite planet/forest2-b
+		distance 485.62
+		period 207.639
+	object
+		sprite planet/rock11-b
+		distance 689.03
+		period 350.931
+	object
+		sprite planet/gas12-b
+		distance 2352.59
+		period 2214.03
+
+system Ibvicklidit-fourelia
+	pos 137.694 -127.065
+	government Xapleaux
+	link Ticska-hearaimiumer
+	asteroids "small rock" 2 3.04
+	asteroids "medium rock" 2 3.116
+	asteroids "large rock" 1 4.674
+	asteroids "medium metal" 1 4.788
+	asteroids "large metal" 1 3.648
+	haze _menu/haze-fullbright
+	fleet "Xapleaux Small Security" 1200
+	hazard "Ember Waste Base Heat" 1
```

Comment:
```suggestion
	hazard "Ember Waste Base Heat" 100
```
Otherwise, these systems which use EWBH invariably softlock every ship that enters them.

Reply:
I spoke with him, it's intended for this space to be dangerous and requiring a special outfit to go there
but yeah it should be an event or smth that only lasts so long

Reply:
Even then, this is probably too extreme.

10 or even 25 would probably be plenty for that.

"Dangerous" and "completely locks out anyone even if they try to prepare" are still different things ;)

Reply:
I brought as many korath heat shunts as I could pack onto the ES, -19k heat at rest, and it was still no match for the heating
I think a frequency of 10 or 20 would do just fine as a very intense heat that (only) threatens to softlock you (but still leaves room for you to recover and escape) unless you're prepared

Reply:
It might even make sense to have a dialog popup the first time the player enters the system to warn them of the heat (like what is done for the nanobots).

Reply:
Yeah this particular one is an accidental fail. XD

Reply:
I'd personally prefer to have a gradual lead in. Have one or two systems leading up to it be quite hot (as in, push the average ship that normally sits at 25% heat up to 50%), then very hot (said average ship is ~75%), before getting into the severe heat.

I kind of disagree with making it completely impassable, though. If someone takes, say, a falcon, and strips out all the weapons to fill it with large heat shunts, it ought to be able to get through the initial periphery; although it could be something like the EW where it's not a  soft-lock, but there's waves of heat that paralyze the player semi-frequently...

Reply:
Yes well, there are notably new systems on the periphery with which to do that. 
Inside the actual defined area of Xapleaux space though, everything is already inside the periphery. There'll be variation within it, and it won't be impassable (unless you're running a hot stock Korath ship or something), but there are only two levels of heat in the core itself though and they're both hot.
A falcon that has stripped its weapons for heat shunts should be able to (once I have some brain space to address it) travel safely anywhere in the core. It won't be able to shoot at anything but it'll be able to fly.